### DISCUSSION TITLE
Lyrics cannot be displayed

### DISCUSSION THREAD

**Discussion by JosipPrekratic** [TT-ReBORN/Georgia-ReBORN#51] [Q&A] [answered]
I'm really enjoying listening to my music collection using your skin in FB2K. However, some of my tracks have embeded lyrics but your skin can't display it. I got this message:

Lyrics cannot be displayed.

For %LYRICS% or %UNSYNCED LYRICS% to always
display properly, you must edit LargeFieldsConfig.txt
and comment out or remove those specific entries
under "Field Spam"

-----------------------------------------------------------

Do you know what I must do?

**Comment by silverwings15** [comment]
just follow the instruction. find LargeFieldsConfig.txt in either %appdata%\foobar2000 (if you used standard install method) or in the profile folder in your portable installation and edit the last two lines (39 and 40) by deleting them or adding // to the front

**Reply by JosipPrekratic** [reply]
Thank you. I found the file. I tried by deleting last two lines but I still get the same message about lyrics.

**Reply by JosipPrekratic** [reply]
I found the solution. I removed last two entries and add it to fieldBasic.

**Reply by JosipPrekratic** [reply]
UPDATE: doesn't work on few songs. :/

**Reply by TT-ReBORN** [reply]
In what tags are the lyrics on those few songs that are not displaying the lyrics? If there are in different tags, you could try to add them in the list of basic meta fields. You could also try to increase the size of basicMetaMax to 4000 and see if it helps...

**Reply by JosipPrekratic** [reply]
Thank you. Will see if this helps.

**Comment by JosipPrekratic** [comment]
I can't find this file in appdata/roaming/foobar2000.

sub, 29. sij 2022. 16:05 silverwings15 ***@***.***> je
napisao:

**Reply by silverwings15** [reply]
then it should be in your foobar directory

**Reply by kbuffington** [reply]
You will probably also have to restart foobar after making the change to LargeFieldsConfig.txt. I think it's only checked at startup.

**Comment by SoWhat84** [comment]
I have the same problem. I edited the LargeFieldsConfig file by removing the last two lines. Unfortunately the error still appears. What else can I do?

**Reply by TT-ReBORN** [reply]
Hi @SoWhat84,

So you've removed in LargeFieldsConfig.txt:

```
fieldSpam=lyrics
fieldSpam=unsynced lyrics
```

from # List of spam meta fields.

But did you try to put: 

```
fieldBasic=lyrics
fieldBasic=unsynced lyrics
```

in # List of basic meta fields?

You could also try to increase the size of basicMetaMax to 4000, save LargeFieldsConfig.txt and restart foobar...

-TT

**Reply by Kenzie1000000** [reply]
Tried both of those and it's still not working

**Reply by TT-ReBORN** [reply, marked as answer]
When you want to read embedded lyrics from the music file, e.g <LYRICS> tag, this is working and it should look like this:

```
# Any changes in this file will take effect on next foobar2000 startup.
# Note, size limit values are in UTF-8 bytes, not characters!
# Note, field names can be enclosed in quotation marks to include trailing spaces etc.
# Size limit for meta fields that appear in the basic fields list
basicMetaMax=2000
# Size limit for meta fields that do not appear in either list
defaultMetaMax=1000
# Size limit for tech info fields
infoMax=200

# List of basic meta fields - essential info such as artist & title
fieldBasic=album
fieldBasic=album artist
fieldBasic=artist
fieldBasic=composer
fieldBasic=conductor
fieldBasic=date
fieldBasic=discnumber
fieldBasic=genre
fieldBasic=keywords
fieldBasic=performer
fieldBasic=producer
fieldBasic=style
fieldBasic=title
fieldBasic=totaldiscs
fieldBasic=totaltracks
fieldBasic=tracknumber
fieldBasic=lyrics
fieldBasic=unsynced lyrics

# List of spam meta fields - rarely useful stuff that *never* gets cached
fieldSpam=accurate rip
fieldSpam=aucdtect
fieldSpam=biography
fieldSpam=cuesheet
fieldSpam=eac logfile
fieldSpam=itunes_cddb_1
fieldSpam=itunmovi
fieldSpam=log
fieldSpam=logfile
```

As previously mentioned you can also try to increase the values for basicMetaMax, defaultMetaMax, infoMax.

If the music file does contain other lyric tags, you can also try to include them in the 
`# List of basic meta fields - essential info such as artist & title`

Once done, you also need to restart foobar to take effect.

-TT

**Reply by almlyxr** [reply]
great. This really works, thanks a lot!
